crates/zed/src/languages/haskell/injections.scm

```diff
@@ -0,0 +1,62 @@
+;; Copyright 2022 nvim-treesitter
```

Comment:
I wonder if it's worth it to just remove `injections.scm` from the PR for now since [`tree-sitter-haskell`](https://github.com/tree-sitter/tree-sitter-haskell) does not actually provide an `injections.scm` query file, so we're now mixing `tree-sitter-haskell` with `nvim-treesitter` 🤔

Reply:
Yea, if `tree-sitter-haskell` doesn't support injections then we might as well remove `injections.scm` here.

Reply:
I think keeping the injections are OK, especially if these are well known.